### DISCUSSION TITLE
"Stuck" tasks, how to clear?

### DISCUSSION THREAD

**Discussion by linsomniac** [semaphoreui/semaphore#1384] [General]
I had an "apt update" job running under Semaphore, which did an update of docker on the semaphore host, which caused Semaphore to restart (I'm using the docker compose method).  So now I have a job in "Running" state, and can't submit any more jobs.  Even restarting Semaphore hasn't cleared that.

Ideally there probably needs to be a check on start-up that clears any running jobs.  I'll submit a ticket on that.

How can I clear them in the mean time?

**Comment by ansibleguy** [comment]
Seems to be a duplicate of: https://github.com/ansible-semaphore/semaphore/issues/1376

**Comment by linsomniac** [comment]
In the interim, I set up an "@reboot" cron job:

```
@reboot root mysql --user=semaphore --host=localhost --port=3306 --protocol=TCP --execute="update task set status = 'error' where status = 'running';" semaphore >/tmp/semaphore-clean.out 2>*1
```

After I put this in /root/.my.cnf:

```
[client]
password="mysemaphoredatabasepassword"
```

**Comment by tboerger** [comment]
Please subscribe to https://github.com/ansible-semaphore/semaphore/issues/1376 if you want to receive an update for this.
